fc/src/main.cpp

```diff
@@ -58,7 +58,7 @@ void publishMessage() {
     DEBUG_SERIAL();
     
     if(DEBUG_MEM){
-        DEBUG_SERIAL("\nFREE RAM: " + String(System.freeMemory()) + "B / 80000B");
+        DEBUG_SERIAL("\nFREE RAM: " + String(System.freeMemory()) + "B / 128000B");
```

Comment:
This is super interesting, I wonder why that's the case? A cursory search through their changelogs doesn't seem to indicate anything (despite this being a _huge_ change) so I wonder if it's just a display issue.

Can we make a `constants.hpp` header in our common library that we can store this value into, so we don't have to change this magic value across all vehicles?

Reply:
I was also quite curious about this and looked through changelogs, but I couldn't find a thing either. There's apparently 256KB of RAM on the processor, so whatever isn't available to the program must be being used for the cell radio and other built-in functions of the Boron. I assume it must have been optimized.

Also, agreed about the constants. There's a lot of things that are shared between the three vehicles, and a lot of duplicate code currently. I think we'll have to look at creating a vehicle class that can be instantiated differently for each of the three "projects". I'll make a separate issue about this.